<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWidget</class>
 <widget class="QWidget" name="LoginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>240</width>
    <height>120</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #F7F9FE</string>
  </property>
  <widget class="QLabel" name="Heading">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Staff</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameField">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">padding: 0 20px;
background: #E0E3E8;
border-bottom: 1px solid #41474D</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Enter Username...</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="passcodeField">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">padding: 0 20px;
background: #E0E3E8;
border-bottom: 1px solid #41474D</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Enter Passcode...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>100</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
border-radius: 17px;
color: #ffffff;
background: #256489</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="font">
    <font>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="toolTip">
    <string>Close Window</string>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/Default/Icons/Res/close-line-icon.svg</normaloff>:/Default/Icons/Res/close-line-icon.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>247</red>
         <green>249</green>
         <blue>254</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>247</red>
         <green>249</green>
         <blue>254</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>247</red>
         <green>249</green>
         <blue>254</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>247</red>
         <green>249</green>
         <blue>254</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>247</red>
         <green>249</green>
         <blue>254</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>247</red>
         <green>249</green>
         <blue>254</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>247</red>
         <green>249</green>
         <blue>254</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>247</red>
         <green>249</green>
         <blue>254</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>247</red>
         <green>249</green>
         <blue>254</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/Default/Icons/Res/close.svg);
border: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
